src/test/java/com/fasterxml/jackson/dataformat/xml/ser/XmlPrettyPrinterTest.java

```diff
@@ -188,4 +189,76 @@ public void testMultiLevel172() throws Exception
                 +"</Company>\n",
                 xml);
     }
+
+    public void testLineFeed_withCustomLineFeed() throws Exception {
+        Company root = new Company();
+        root.employee.add(new Employee("abc"));
+
+        String xml = _xmlMapper.writer()
+            .with(new DefaultXmlPrettyPrinter().withCustomLineFeed("\n\rLF\n\r"))
+            .with(ToXmlGenerator.Feature.WRITE_XML_DECLARATION)
+            .writeValueAsString(root);
+        // unify possible apostrophes to quotes
+        xml = a2q(xml);
+
+        // with indentation, should get linefeeds in prolog/epilog too
+        assertEquals("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```

Comment:
I am not sure if this line should also end with customLineFeed?

Reply:
see the answer to your follow up question